src/Skynet.Server/Services/DeliveryService.cs

```diff
@@ -39,47 +40,39 @@ internal sealed class DeliveryService
         }
 
         #region packet and message broadcast
-        public async Task<IReadOnlyList<Task>> SendToAccount(Packet packet, long accountId, IClient exclude)
+        public async Task StartSendToAccount(Packet packet, long accountId, IClient exclude)
```

Comment:
iirc, the naming convention is `BeginSendToAccount` instead of Start

Reply:
I wanted to avoid confusion with [APM](https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/asynchronous-programming-model-apm) methods like `BeginSendToAccount` and `EndSendToAccount`. PowerShell uses the verb `Start` to start operations without waiting for them to complete.